guides/common/modules/con_overview-of-recommended-disaster-recovery-plans.adoc

```diff
@@ -0,0 +1,76 @@
+To recover {Project} services, switch DNS records of the active {ProjectServer} with the passive {ProjectServer}.
+This ensures that the passive server becomes the active server.
+All hosts remain connected without configuration updates.
+Disadvantages and expected impact:::
+If the network attached storage is replicated to another location, expect some amount of data inconsistency after recovery based on the synchronization interval.
+
+Active and passive {ProjectServer}, with backup and restore::
+How do I back up?:::
+Ensure periodic backups of your {ProjectServer}.
+Copy this backup to a passive {ProjectServer} and restore them on the passive server.
+How will I recover in case of a disruptive event?:::
+To recover {Project} services, switch DNS records of the active {ProjectServer} with the passive {ProjectServer}.
+This ensures that the passive server becomes the active server.
+All hosts remain connected without configuration updates.
+//Use a low DNS time to live (TTL) value to help ensure that hosts reach the new active {ProjectServer} quickly.
+//Consider your tolerance for how long it takes before your hosts are able to reconnect and access the correct {ProjectServer} and set your TTL according to your needs.
+Disadvantages and expected impact:::
+Expect some amount of data inconsistency after recovery, based on how often you took and restored backups and on how long it takes to complete the restore process.
+
+Dual active {ProjectServer}::
+How do I back up?:::
+Operate an active, independent {ProjectServer} per data center.
+Hosts from each data center are registered to the {ProjectServer} in that data center.
+Then configure automation to ensure recovery in case of a disruptive event.
+For example, you can periodically run a health check and if the health check discovers that the current {ProjectServer} a host is registered to does not resolve, the host is re-registered to the other {ProjectServer}.
++
+To minimize downtime, you can automate the recovery in various ways.
+For example, you can use the {Project} Ansible collection.
+For more information, see {AdministeringDocURL}Managing_Project_with_Ansible_Collections_admin[Managing {Project} with Ansible collections] in _{AdministeringDocTitle}_.
+How will I recover in case of a disruptive event?:::
+To recover {Project} services, re-configure hosts to point at the {ProjectServer} in the other data center.
+You will need to re-register each host to the new server.
+Disadvantages and expected impact:::
+ifdef::katello,orcharhino,satellite[]
+You must ensure that content synchronization and content view creation are synchronized to create the same content view in each {Project} and prevent content drift.
+Content drift occurs when available content deviates from the intended state defined by a content view.
+If you fail to prevent content drift, expect inconsistency in the content that is available to hosts.
+
+//As an alternative, you can implement the following setups:
```

Comment:
Please also remove or explain what's still TODO.

Reply:
Removed the comment.